build/template_kql/common/extend_resource.kql

```diff
@@ -146,10 +145,9 @@ resources
     (type == 'microsoft.web/sites'), "ASP AZ config",
     (type == 'microsoft.web/serverfarms'), iff(properties.zoneRedundant == 'true', 'Configured', 'Not Configured'),
     (type == 'microsoft.sql/servers/databases' and sku.tier <> 'DataWarehouse'), case( //SQL AZ SKU & region availability filter
-    sku.name == 'Basic' or sku.name == 'Standard' or sku.name == 'System', 'Not Applicable',
-    sku.name == 'GeneralPurpose' and location !in~ ('westeurope', 'northeurope', 'westus2', 'francecentral', 'eastus', 'eastus2', 'southeastasia', 'qatarcentral', 'australiaeast', 'japaneast', 'uksouth'), 'Not Applicable',
```

Comment:
Need to ensure whether we validate the regions supported for GP tier: https://learn.microsoft.com/en-us/azure/reliability/migrate-sql-database?tabs=portal#prerequisites

Reply:
I updated about this in this commit.
https://github.com/Azure/reliability-workbook/pull/153/commits/ac51f7b9fb1e936fc2d16330591f5cf8587b4999